.readthedocs.yml

```diff
@@ -6,9 +6,13 @@ version: 2
 # https://github.com/aiidateam/aiida-core/issues/1472
 formats: []
 
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+
```

Comment:
Is this necessary? Is Python 3.9 or 3.10 not available on the default Ubuntu that is run on RTD? Would simply changing line 11 to `version: 3.10` not work?

Reply:
Never mind. Apparently the `python.version` option is deprecated: https://docs.readthedocs.io/en/stable/config-file/v2.html#python-version

Reply:
>Would simply changing line 11 to version: 3.10 not work?

Since this is the lazy man's option, you can safely assume that is the first thing I tried. 😁